pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -567,7 +567,14 @@ func (r *ReconcileIntrusionDetection) Reconcile(ctx context.Context, request rec
 		return reconcile.Result{}, err
 	}
 
-	typhaNodeTLS.TrustedBundle.AddCertificates(esgwCertificate)
+	typhaNodeTLS.TrustedBundle.AddCertificates(linseedCertificate)
```

Comment:
I don't understand why the TyphaNodeTLS trusted bundle needs to have the Linseed certificate (or the EGW certificate) - do you?

Reply:
DeepPacketInspection queries Typha to get updates for resources like new workload endpoints. Most likely, the esgateway cert was bundled up together.